pkg/ds/farm.go

```diff
@@ -564,6 +565,25 @@ func (dsf *Farm) spawnDaemonSet(
 
 	desiresCh := ds.WatchDesires(quitSpawnCh, updatedCh, deletedCh)
 
+	go func() {
+		aggregateHealth := replication.AggregateHealth(ds.PodID(), *dsf.healthChecker)
+		ticks := time.NewTicker(1 * time.Minute)
+
+		defer aggregateHealth.Stop()
```

Comment:
Excuse me, what? This line panics on closing a nil channel... the `quit` channel of `replication.podHealth` is NEVER set, which means the health aggregator is unstoppable. This can't be right.

This implies we were never stopping any of our health aggregators??? We start one for every replication, and never clean them up, so the number of watches increases monotonically over time???